code/index.html

```diff
@@ -20,25 +20,38 @@
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the character have</h1>
 
       <select id="questions">
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+          <option value="pink">pink hair</option>
+          <option value="black">black hair</option>
+          <option value="grey">grey hair</option>
+          <option value="hidden">hidden hair</option>
           <!-- Add the rest of hair colors as options  -->
```

Comment:
I believe these comments can be removed before publishing, since they're more of a help when programming.